### PR TITLE
feat(input): add input documentation

### PR DESCRIPTION
**Pull request by abdelazizsakhi**

### CODE REVIEW

**Review by flowg — CHANGES_REQUESTED** [review]

File: apps/playground/src/app/modules/components/input-module/input-module.component.html [hunk 1]
```diff
@@ -1,94 +1,225 @@
 <section class="module-description">
-  <h2>Description</h2>
-  <p>Some description</p>
+  <h2>DsfrInputComponent</h2>
+  <p>
+    The <code>DsfrInputComponent</code> purpose is to represent the input form field
```
[the remark this answers is not in the document]
Comment (flowg) [reply apps/playground/src/app/modules/components/input-module/input-module.component.html:4]: Not done

**Review by flowg — CHANGES_REQUESTED** [review]

File: apps/playground/src/app/modules/components/input-module/input-module.component.ts [hunk 2]
```diff
@@ -42,6 +64,39 @@ export class InputModuleComponent implements OnInit {
 
     this.form = this.formBuilder.group({
       input: ''
-    })
+    });
+
+
+    this.handleErrors('Please, enter a label for this Component', 'label');
+    this.handleErrors('Please, enter a failure meessage for this Component', 'failureMessage');
```
[the remark this answers is not in the document]
Comment (abdelazizsakhi) [reply apps/playground/src/app/modules/components/input-module/input-module.component.ts]: Can we fix this together, the component reacts very strangely

File: apps/playground/src/app/modules/components/input-module/input-module.component.ts [hunk 3]
```diff
@@ -1,30 +1,102 @@
...
-      label: 'DSFR Input works 😁'
-    });
-    this.inputForm = this.formBuilder.group({
-      trytest: ['', Validators.required]
+      input: ''
     });
+
+    this.handleErrors('Please, enter a label for this Component', 'label');
+    this.handleErrors(
+      'Please, enter a failure meessage for this Component if the hasFailed attribute is true'
```
Comment (flowg) [inline apps/playground/src/app/modules/components/input-module/input-module.component.ts:65]: `failure message`

File: apps/playground/src/app/modules/components/input-module/input-module.component.ts [hunk 4]
```diff
@@ -1,30 +1,102 @@
...
   templateUrl: './input-module.component.html',
   styleUrls: ['./input-module.component.scss']
 })
-export class InputModuleComponent implements OnInit {
+export class InputModuleComponent implements OnInit, OnDestroy {
 
+  private unsubscribe$ = new Subject<void>();
+
+  inputForm = this.formBuilder.group({
+    label: ['This is a label', Validators.required],
```
Comment (flowg) [inline apps/playground/src/app/modules/components/input-module/input-module.component.ts:27]: Here, only `label` needs to be `required`

**Review by flowg — CHANGES_REQUESTED** [review]

**Review by flowg — APPROVED** [review]

**Pull request merged**